lib/public/components/common/clipboardTarget.js

```diff
@@ -14,12 +14,13 @@
 import { h } from '/js/src/index.js';
 
 /**
- * Copies a given text to the clipboard if the window is secure
+ * Copies a given text to the clipboard if the window is secure. Wraps child components.
  * @param {copyText} copyText the text to copy
+ * @param {string} parentSelector the css selector of the parent node e.g. "button.btn"
  * @param {Component} children components
  * @return {Component} the children components wrapped in the clipboardTarget
  */
-export const clipboardTarget = (copyText, children) => h('a', { onclick: (e) => {
+export const clipboardTarget = (copyText, parentSelector, children) => h(parentSelector, { onclick: (e) => {
```

Comment:
Here I think we should not mix the behavior-based component (copy to clipboard) from actual dom. Plus, taking a mithril vnode description as argument makes the code completely dependent on the mithril syntax, and blocks us if for example at some point we use jsx.
Could you please try to remove the 'parentSelector' and simply wrap children in a div?
The `button.btn` should simply be included in the children.